test/TestConfigurationData.py

```diff
@@ -0,0 +1,17 @@
+
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.StandardSequences.Eras import eras
```

Comment:
It's a CMS thing :
https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideCmsDriverEras

2015-09-07 13:07 GMT+02:00 Sébastien Wertz <notifications@github.com>:

> In test/TestConfigurationData.py
> <https://github.com/cp3-llbb/Framework/pull/32#discussion_r38853486>:
>
> > @@ -0,0 +1,17 @@
> > +
> > +import FWCore.ParameterSet.Config as cms
> > +
> > +from Configuration.StandardSequences.Eras import eras
>
> Hi, is that a CMS thing or is a file missing from the commit?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cp3-llbb/Framework/pull/32/files#r38853486>.
>



-- 
Sébastien Brochet